.gitignore

```diff
@@ -4,7 +4,6 @@ bitlbee
 user-guide.txt
 user-guide.html
 help.txt
-debian
```

Comment:
This was intentional, so that only explicitly added files are tracked by git (and excluding all the automatically generated stuff). How does it improve things for you?